src/ui/components/DropdownMenuItem/styles.scss

```diff
@@ -33,6 +33,13 @@
   margin-top: 8px;
 }
 
+.DropdownMenuItem--disabled {
+  &,
+  & button {
+    @include disabled();
```

Comment:
Could you check for accessibility issues with this text (especially on hover) using something like https://color.review/ ? It looks like low contrast to me, meaning that it could be hard to read.

Reply:
ah, that's true. I increased the opacity so that the contrast is better and we obtain good results on color.review.